odml/dtypes.py

```diff
@@ -28,6 +28,17 @@ class DType(str, Enum):
     def __str__(self):
         return self.name
 
+default_values = {
+    'string': '',
+    'text': '',
+    'int': 0,
+    'float': 0.0,
+    'url': '',
+    'datetime': '2015-06-20 10:00:00',
+    'date': '2015-06-20',
```

Comment:
Where did these default values come from?
Can we do `now` somehow?

Reply:
You mean for the `datetime`, `date`, and `time` types, right ? 
Yeah, we can implement something like `now` for them. I'll make the changes. Maybe we can have it as a function ?